src/obstacle_avoidance/include/MyNode.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * Created By: Chris Heathe
+ * Created On:  October28, 2017
+ * Description: A node that will theoretically interpret lidar data,
+ *              and help Olaf navigate through a hole in a row of cones
+ *              Stress on *theoretically*
+ */
+
+#ifndef obstacle_avoidance_MYNODE_H
+#define obstacle_avoidance_MYNODE_H
+
+#include <iostream>
+#include <std_msgs/String.h>
+#include <sensor_msgs/LaserScan.h>
+#include <ros/ros.h>
+#include <sb_utils.h>
+#include <geometry_msgs/Twist.h>
+
+class MyClass {
+public:
+    MyClass(int argc, char **argv, std::string node_name);
+    /**
+     * Adds an exclamation point to a given string
+     *
+     * Some Longer explanation should go here
+     *
+     * @param input_string the string to add an exclamation point to
+     *
+     * @return input_string with an exclamation point added to it
+     */
+     static std::string addCharacterToString(std::string input_string, std::string suffix);
+     std::string suffix;
```

Comment:
We can probably remove both the `addCharacterToString` function and the `suffix` field here, since they're remnants from the sample node.